metabase/metabase
# dx: handle CSP errors in pivot table

Related to https://github.com/metabase/metabase/issues/45698

### Description

Pivot Table component uses `AutoSizer` which adds styles in runtime, that thing breaks CSP

### How to verify

open app in dev mode, go to the dashboard with a question with pivot table, make sure there is no CSP error
in devtools 

### Demo

### Before


https://github.com/user-attachments/assets/0cf5ec5c-cbd9-4f79-b572-3ebc5a37b768

### After


https://github.com/user-attachments/assets/f92971d2-124e-4349-a054-3d93856e13ee
